tests/monotouch-test/SystemConfiguration/CaptiveNetworkTest.cs

```diff
@@ -82,16 +83,27 @@ public void TryCopyCurrentNetworkInfo ()
 #endif
 		}
 
+		[DllImport (Constants.SystemConfigurationLibrary)]
+		extern static IntPtr /* CFArrayRef __nullable */ CNCopySupportedInterfaces ();
+
 		[Test]
 		public void TryGetSupportedInterfaces ()
 		{
 			StatusCode status;
 #if __TVOS__
 			Assert.Throws<NotSupportedException> (() => CaptiveNetwork.TryGetSupportedInterfaces (out var ifaces));
 #else
+
 			status = CaptiveNetwork.TryGetSupportedInterfaces (out var ifaces);
 			Assert.AreEqual (StatusCode.OK, status, "Status");
-			Assert.IsNull (ifaces, "Null Interfaces");
```

Comment:
I think you can just remove this line (and not add the other code), because the added test code just does the same thing the `CaptiveNetwork.TryGetSupportedInterfaces` method does, so there's not much testing value.